tools/sggolicenses/tools.go

```diff
@@ -57,9 +53,45 @@ func CheckDir(ctx context.Context, directory string, disallowedTypes ...string)
 	return cmd.Run()
 }
 
-// Check for disallowed types of Go licenses.
+// Check for disallowed types of Go licenses in a specific directory.
 // By default, Google's forbidden and restricted types are disallowed.
-func Check(ctx context.Context, disallowedTypes ...string) error {
+func CheckDir(ctx context.Context, directory string, disallowedTypes ...string) error {
+	var appendArgs []string
+	if len(disallowedTypes) > 0 {
+		appendArgs = append(appendArgs, "--disallowed_types="+strings.Join(disallowedTypes, ","))
+	} else {
+		appendArgs = append(appendArgs, "--disallowed_types=forbidden,restricted")
+	}
+
+	return checkDir(ctx, directory, appendArgs)
+}
+
+// Check for disallowed types of Go licenses in a specific directory.
+// By default, Google's forbidden and restricted types are disallowed.
+// Allows to ignore some dependencies by specifying their package path (ie "github.com/einride/sage").
+func CheckDirWithIgnored(
```

Comment:
with this many helpers it might be better to inline it in the specific repos where it's used - the code here is just meant to be the "most common default" use case - not all possible use cases

Reply:
will close for now, the problem in the repo has been solved in another way